modules/globebrowsing/video.asset

```diff
@@ -0,0 +1,28 @@
+local layer = {
+  Identifier = "TestVideo",
+  File = "C:/Users/ylvaselling/Downloads/equirectangular-projection-testmovie02.mp4",
```

Comment:
OBS! Local file that should be removed

Also, this asset should not be included in this PR